MapboxDirections/MBSpeedLimit.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  MBSpeedLimit.swift
+//  MapboxDirections
+//
+//  Created by Avi Cieplinski on 4/15/19.
+//  Copyright © 2019 Mapbox. All rights reserved.
+//
+
+import Foundation
+
+@objc(MBSpeedLimit)
```

Comment:
This class needs a documentation comment; otherwise, the entire class will be missing from the generated jazzy docset.